test/test.js

```diff
@@ -69,33 +69,30 @@ describe('onclickoutside hoc', function() {
 
 
   it('should call the specified handler when clicking the document', function() {
-    var Component = React.createClass({
-      getInitialState: function() {
-        return {
+    class Component extends React.Component {
```

Comment:
@Pomax A stateless component might be sufficient to generate the error, but I do think that testing the 3 ways to create a component (stateful component with inheritance, createClass and stateless function) will give you the best chance to catch these errors early next time.

I should be able to add those tests by the end of the day. If you prefer that I only add tests for stateless vs createClass, let me know.